pkg/daemon/ceph/agent/agent.go

```diff
@@ -77,12 +78,21 @@ func (a *Agent) Run() error {
 	}
 
 	flexDriverVendors := []string{flexvolume.FlexvolumeVendor, flexvolume.FlexvolumeVendorLegacy}
-	for _, vendor := range flexDriverVendors {
+	for i, vendor := range flexDriverVendors {
+		if i > 0 {
+			// Wait before the next driver is registered. in 1.11 and newer there is a timing issue if flex drivers are registered too quickly.
+			// See https://github.com/rook/rook/issues/2064
```

Comment:
I have looked at the PR comment that is linked in the issue description, though is that something that should also be tracked at Kubernetes (even though it as you wrote seems to "only" appear in our CI)?

Reply:
Actually, looks like others had hit it according to #1501, which references a k8s issue. I'll link that instead.